identity/client/src/pages/login/LoginPage.tsx

```diff
@@ -169,8 +171,10 @@ export const LoginPage: React.FC = () => {
   });
 
   const redirectUrl = getRedirectUrl(location.search);
+  const contextPath = window.clientConfig?.contextPath ?? "";
   const onSuccess = useCallback(() => {
-    window.location.href = redirectUrl ?? `/identity${Paths.users()}`;
+    window.location.href =
+      redirectUrl ?? mergePathname(contextPath, `/identity${Paths.users()}`);
```

Comment:
Why do you need the contextPath here?
Can't you just call the `getBasePathBeforeIdentity() + /identity${Paths.users()}`?
This way, we would not need `mergePathname` utility anymore. I guess you also do not need the backend logic to expose the contextPath.